packages/larva-patterns/components/c-button/c-button.prototype.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
 	"c_button_classes": "lrv-a-unstyle-button lrv-u-cursor-pointer",
+	"c_button_target_attr": "_self",
```

Comment:
What about leaving this empty instead of adding `_self`?

Reply:
`_self` is the default behavior, We can keep empty as well.

Reply:
Right - I would rather let the browser decide default behavior just in case that changes (small chance, but still), so let's leave it blank.

Reply:
Removed default value.